lib/Cleantalk/ApbctWP/State.php

```diff
@@ -144,6 +141,7 @@ class State extends \Cleantalk\Common\State
         'last_remote_call'               => 0, //Timestam of last remote call
         'current_settings_template_id'   => null,  // Loaded settings template id
         'current_settings_template_name' => null,  // Loaded settings template name
+        'ajax_type'                      => false, // Ajax type
```

Comment:
If you will using this "data" please, add it to updater script for a new version like this. It considers any new data|settings|network things.

`global $apbct;`
`$apbct->data['ajax_type'] = false;`
`$apbct->save('data');`

Reply:
Ready